snowpack/assets/require-or-import.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
```

Comment:
This file is explicitly `require`d inside of TypeScript so that it is not compiled.